src/objects.js

```diff
@@ -147,3 +147,61 @@ export class Page {
     return new Page(this.data.map(func), this.before, this.after)
   }
 }
+
+/** FaunaDB time. See the [docs](https://faunadb.com/documentation/queries#values-special_types). */
+export class FaunaTime {
+  /**
+   * @param {string|Date} value If a Date, this is converted to a string.
+   */
+  constructor(value) {
+    if (value instanceof Date)
+      value = value.toISOString()
```

Comment:
i seem to recall a conversation about this somewhere. In the Java/scala client, I store the native date/time types, and then convert them to string only at serialization time. Should we do that here?